vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -178,5 +188,16 @@ def parse_base_item(item)
         :resource_version   => item.metadata.resourceVersion
       }
     end
+
+    HUMAN_SIZE_SUFFIXES = ["Ki", "Mi", "Gi", "Ti"]
```

Comment:
It's a common mistake to believe that K/M/G/T are powers of 1024, instead they are powers of 1000. Kubernetes uses the correct suffixes Ki/Mi/Gi/Ti (etc.) that are instead more precisely identifying powers of 1024. More info here: http://en.wikipedia.org/wiki/Binary_prefix

Anyway I will re-send this patch fixing rubocop warnings.